ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1382,12 +1382,45 @@ literals:
   - ON
   - OFF
 
+# Addition of telemetry for UPS battery monitoring
+- battery_status:
+  - ACTIVE
+  - INACTIVE
+- bypass_status:
+  - ACTIVE
+  - INACTIVE
+- inverter_mode:
+  - SYNCHRONOUS
+  - ASYNCHRONOUS
+- overload_alarm:
+  - ACTIVE
+  - INACTIVE
+- bypass_phase_status:
```

Comment:
what is this for? not understanding from the field name what its usage should be.

Reply:
im okay with this.